packages/react-scripts/config/get-custom-config.js

```diff
@@ -1,21 +1,26 @@
-var customizers = require('./customizers');
+const customizers = require('./customizers');
 
 function getCustomConfig(prod) {
-  var prod = prod || false;
-  var env = env || {};
-  var result = Object
-    .keys(customizers)
-    .reduce(function (finalConfig, customizerKey) {
-      var customizer = customizers[customizerKey];
+  prod = prod || false;
+  const env = {};
+
+  const customizerKeys = customizers.keys();
```

Comment:
Doesn't this need to be the following?

```
const customizerKeys = Object.keys(customizers);
```

I tried out your fork here, and it failed on that line. I fixed it in karlhorky/custom-react-scripts#3ad34e8930dc4d72c68622714c910c16a6f4b27d